src/App/Command/UpdateCommand.php

```diff
@@ -59,7 +59,7 @@ protected function beforeProcessingPackages(InputInterface $input): void
 
     protected function afterProcessingPackages(): void
     {
-        $this->packageService->createSymbolicLinks($this->getPackageList(), $this->getIO());
+        $this->packageService->createSymbolicLinks($this->getTargetPackages(), $this->getIO());
```

Comment:
In process this command runned `composer update` and may be that need add symbolic links don't only to target packages.

Reply:
In the process of running the `composer update` command, update all symbolic links for the target packages,

`$this->packageService->createSymbolicLinks(
    $this->getTargetPackages(),
     $this->getPackageList()->getInstalledAndEnabledPackages(),
    $this->getIO()
 );`
 
 or remove links to the target packages from the target packages and then update, instead of the list of packages?
 
` $this->packageService->removeSymbolicLinks($package, $this->targetPackages(), $io);`

Reply:
I meant `update` command of Yii Dev Tool that run `composer update` also:
https://github.com/yiisoft/yii-dev-tool/blob/master/src/App/Command/UpdateCommand.php#L85

Reply:
Were able to resolve the issues mentioned in this pull request?

https://github.com/yiisoft/yii-dev-tool/blob/fd345f116b915a81e32c4b2302309230e147e919/src/App/PackageService.php#L160-L166
I noticed the problem of creating symbolic links when working with fork repositories.
Checking fork package instead of upstream repository